gulpfile.js

```diff
@@ -0,0 +1,46 @@
+let gulp = require('gulp'),
```

Comment:
'let' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).